contrib/src/main/java/com/datatorrent/contrib/couchbase/AbstractCouchBaseInputOperator.java

```diff
@@ -149,15 +145,8 @@ public void partitioned(Map<Integer, Partition<AbstractCouchBaseInputOperator<T>
     int numPartitions = conf.getServers().size();
     List<String> list = conf.getServers();
     Collection<Partition<AbstractCouchBaseInputOperator<T>>> newPartitions = Lists.newArrayListWithExpectedSize(numPartitions);
-    Kryo kryo = new Kryo();
     for (int i = 0; i < numPartitions; i++) {
-      ByteArrayOutputStream bos = new ByteArrayOutputStream();
-      Output output = new Output(bos);
-      kryo.writeObject(output, this);
-      output.close();
-      Input lInput = new Input(bos.toByteArray());
-      @SuppressWarnings("unchecked")
-      AbstractCouchBaseInputOperator<T> oper = kryo.readObject(lInput, this.getClass());
+      AbstractCouchBaseInputOperator<T> oper = PartitionerUtils.cloneOperatorByKryo(this);
```

Comment:
I've created a threadlocal pool for kryo objects. I think it's even better idea to create kryo on the fly.